inference-engine/thirdparty/clDNN/src/graph_optimizer/remove_redundant_reorders.cpp

```diff
@@ -363,6 +363,59 @@ void remove_redundant_reorders::run(program_impl& p) {
         p.remove_if_dangling(*node);
     }
 
+    // This pass removes reorder for Convolution b_fs_yx_fsv16 -> bfyx
+    if (enable_reorder_fusing) {
+        itr = p.get_processing_order().begin();
+        while (itr != p.get_processing_order().end()) {
```

Comment:
Could you merge this nodes traverse with another one?

Reply:
With another one you mean as conv bfyx -> fsv reorder ? At first we tried that, but the conditions are not pretty synchronized b/w those two so that we needed many conditions in the loop and it was not comprehensive too..

Reply:
I think it can be something as follows
```cpp
auto try_fuse_reorder_fsv16_to_bfyx = [&](reorder_node& node) {/* impl */};
auto try_fuse_reorder_bfyx_to_fsv32= [&](reorder_node& node) {/* impl */};
itr = p.get_processing_order().begin();
while (itr != p.get_processing_order().end()) {
    if (!node->is_type<reorder>())
        continue;

    auto& r_node = node->as<reorder>();

    // common checks here

    try_fuse_reorder_fsv16_to_bfyx(r_node);
    try_fuse_reorder_bfyx_to_fsv32(r_node);
}
```

I don't insist on that, but extra graph traverse may impact load time, so ideally we need to merge them when possible.

Reply:
That can be applicable. @kelvinchoi-intel  Could you please apply vladimir's suggestion?

Reply:
Okay, I will try Vladimir's suggestion.

Reply:
Applied